docs/network/operators.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Network operators
+---
+
+Etherlink relies on operators who run nodes.
+For information on the roles of these different nodes, see [Etherlink architecture](/network/architecture).
+
+## Nodes
+
+Anyone can run Etherlink [EVM nodes](/network/evm-nodes) and [Smart Rollup nodes](/network/smart-rollup-nodes), and due to the optimistic nature of Tezos [Smart Rollups](https://docs.tezos.com/architecture/smart-rollups) it takes only one honest node operator to keep Etherlink secure.
+Honest nodes can catch any misbehavior by other nodes by refuting their incorrect commitments.
+
+These organizations currently run Etherlink nodes:
+
+- [The Tezos Foundation](https://tezos.foundation/)
+- [MIDL.dev](http://midl.dev/)
+- [Zeeve](https://www.zeeve.io)
+
+The sequencer operator also runs a Smart Rollup node that the sequencer uses exclusively.
+
+## Sequencer
+
+Etherlink users rely on the sequencer to put transactions in a fair order and to ensure that transactions are included quickly.
+Under normal circumstances, Etherlink users trust the sequencer to include their transactions, but if it does not, they can insert their transactions into the delayed inbox to force them to be included, as described in [Transaction lifecycle](/network/architecture#transaction-lifecycle).
+
+Currently only the sequencer operator can run a sequencer, which is an Etherlink EVM node running in sequencer mode.
```

Comment:
```suggestion
Currently only the sequencer operator can run a sequencer, which is an Etherlink EVM node running in sequencer mode.
Users relying on the sequencer to include their transactions in Etherlink pay a so-called DA (Data Availibility) fee, which covers the cost of including the transaction in a Layer 1 block.
```

So that the next sentence makes sense.